File: 3k/1s/cpp/conversation/NoFormalist.cpp
class NoFormalist: public Person
{
protected:
    char* getTypeText()
    {
        return "Неформал";
    }

public:
    NoFormalist(char* newName, int newAge)
    {
        init(newName, newAge);
    }

    const void Hello(Person* &personTwo) {
        cout << "Привет, " << personTwo->getName();
    }
};

File: 3k/2s/java/5. Conversation/src/com/company/NoFormalist.java
package com.company;

public class NoFormalist extends Person
{
    public NoFormalist(String newName, int newAge, int newSex)
    {
        super(newName, newAge, newSex);
    }

    public String getTypeText()
    {
        return "Неформал" + getTypeTextSuffix();
    }

    public void hello(Person twoPerson)
    {
        System.out.print("Привет, " + twoPerson.getName());
    }
}
